lib/cfndsl/resources.rb

```diff
@@ -1,13 +1,12 @@
 require 'cfndsl/jsonable'
-require 'cfndsl/metadata'
 require 'cfndsl/properties'
 require 'cfndsl/update_policy'
 
 module CfnDsl
   # Handles Resource objects
   class ResourceDefinition < JSONable
-    dsl_attr_setter :Type, :DependsOn, :DeletionPolicy, :Condition
-    dsl_content_object :Property, :Metadata, :UpdatePolicy, :CreationPolicy
```

Comment:
@kornypoet Do we not want to keep this in for people that might still be using it and deprecate it before a major release? Or has it never been relevant as the metadata has always sat at the top level?